test/unit/full_text_search/change_git_test.rb

```diff
@@ -24,64 +24,102 @@ def test_fetch_changesets
-                       "filemane with spaces.txt",
-                       " filename with a leading space.txt ",
-                       "latin-1/test00.txt",
-                       "README",
-                       "latin-1-dir/make-latin-1-file.rb",
-                       "issue-8857/test00.txt",
-                       "issue-8857/test01.txt",
-                     ],
-                     {
-                       "project_id" => @project.id,
-                       "source_id" => first_change.id,
-                       "source_type_id" => Type.change.id,
-                       "last_modified_at" => parse_time("2007-12-14T09:24:01Z"),
-                       "registered_at" => parse_time("2007-12-14T09:24:01Z"),
-                       "container_id" => repository.id,
-                       "container_type_id" => Type.repository.id,
-                       "title" => "images/edit.png",
-                       "content" => "",
-                       "custom_field_id" => null_number,
-                       "is_private" => null_boolean,
-                       "tag_ids" => [Tag.extension("png").id],
-                     },
-                     {
-                       "project_id" => @project.id,
-                       "source_id" => last_change.id,
-                       "source_type_id" => Type.change.id,
-                       "last_modified_at" => parse_time("2011-01-01T03:00:00Z"),
-                       "registered_at" => parse_time("2011-01-01T03:00:00Z"),
-                       "container_id" => repository.id,
-                       "container_type_id" => Type.repository.id,
-                       "custom_field_id" => null_number,
-                       "title" => "issue-8857/test01.txt",
-                       "content" => <<-CONTENT,
+      expected_first_change = {
+                                "project_id" => @project.id,
+                                "source_id" => first_change.id,
+                                "source_type_id" => Type.change.id,
+                                "last_modified_at" => parse_time("2007-12-14T09:24:01Z"),
+                                "registered_at" => parse_time("2007-12-14T09:24:01Z"),
+                                "container_id" => repository.id,
+                                "container_type_id" => Type.repository.id,
+                                "title" => "images/edit.png",
+                                "content" => "",
+                                "custom_field_id" => null_number,
+                                "is_private" => null_boolean,
+                                "tag_ids" => [Tag.extension("png").id],
+                              }
```

Comment:
インデントを直してもらえますか？
RuboCopを入れたほうがいいですか？

```suggestion
      expected_first_change = {
        "project_id" => @project.id,
                                "source_id" => first_change.id,
                                "source_type_id" => Type.change.id,
                                "last_modified_at" => parse_time("2007-12-14T09:24:01Z"),
                                "registered_at" => parse_time("2007-12-14T09:24:01Z"),
                                "container_id" => repository.id,
                                "container_type_id" => Type.repository.id,
                                "title" => "images/edit.png",
                                "content" => "",
                                "custom_field_id" => null_number,
                                "is_private" => null_boolean,
                                "tag_ids" => [Tag.extension("png").id],
      }
```

Reply:
fix: [6b58c6d](https://github.com/clear-code/redmine_full_text_search/pull/157/commits/6b58c6dec426fd16a3f8087239c8413ac5e0b14f)
RuboCop を入れたいです。変更を入れるファイルから一つずつ反映して行く感じで導入していきたいです。